src/Microsoft.ML/Data/CollectionDataSource.cs

```diff
@@ -52,6 +52,8 @@ public void SetInput(IHostEnvironment environment, Experiment experiment)
                 experiment.SetInput(_dataViewEntryPoint.Data, _dataView);
             }
 
+            public Var<IDataView> GetInputData() => null;
```

Comment:
>public Var<IDataView> GetInputData() => null; [](start = 12, length = 45)

why I don't see this change in TextLoader, it also suppose to implement ILearningPipeline?

Reply:
Its code generated there in CSharpAPI.cs

---
In reply to: [191002526](https://github.com/dotnet/machinelearning/pull/212#discussion_r191002526) [](ancestors = 191002526)